monitor.py

```diff
@@ -6,12 +6,11 @@
 import requests
 from datetime import datetime
 import xml.etree.ElementTree as ET
+from monitoring_config import PROW_VIEW_URL,JOB_LINK_URL,RELEASE_URL
```

Comment:
Please don't import like this. If there are n number of constants, this will be huge.

Reply:
```import monitoring_config

urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
PROW_URL = ""
final_job_list=[]
prow_view_url = monitoring_config.PROW_VIEW_URL
job_link_url=monitoring_config.JOB_LINK_URL
release_url=monitoring_config.RELEASE_URL
```